blackbox/docs/configuration.txt

```diff
@@ -1851,6 +1851,26 @@ to your config. This will prevent allocating primary and replica
 of the same shard on the same machine even if more than one instances
 running on it.
 
+Other System Configuration
+==========================
+
+Virtual Memory
+--------------
+
+CrateDB uses a hybrid ``MMap``/``NIO`` file system storage directory to store
```

Comment:
Yeah, I'm not entirely sure the best way of formatting them. https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-store.html#simplefs doesn't make them literals, so perhaps this is the best approach.